docs/src/_concepts/environments.md

```diff
@@ -69,6 +70,12 @@ That is, `MELTANO_PROJECT_ROOT` and `MELTANO_ENVIRONMENT`.
 In the below example, the `$MELTANO_PROJECT_ROOT/path/to/a/file.json` value will properly read the `$MELTANO_PROJECT_ROOT`
 environment variable and inject the full value as `$MY_ENV_VAR` into the environment.
 
+## `state_id_suffix`
+
+Environments can also define a `state_id_suffix` - a custom suffix used in the generation of a state ID for each extractor/loader pair passed to [`meltano run`]((/reference/command-line-interface#run)).
+
+`state_id_suffix` supports interpolation of environment variables to allow for dynamic state IDs (e.g. unique state for multiple `meltano run` invocations using the same environment and EL pair(s)).
+
 ```yaml
```

Comment:
This section goes with the "The env mapping" section, not with the state id suffix. It needs to be moved back up. Check the preview https://deploy-preview-6758--meltano.netlify.app/concepts/environments#state_id_suffix

Reply:
My bad, fixing now :+1:

Reply:
See https://github.com/meltano/meltano/pull/6758/commits/fdca026b0c5376d23e7d147e19693e3b13003a3d